pages/docs/observability/features/trace-ids-and-distributed-tracing.mdx

```diff
@@ -129,38 +129,49 @@ with langfuse.start_as_current_span(name="outer-operation") as span:
 ```
 
 </Tab>
-<Tab>
+<Tab title="JS/TS SDK">
```

Comment:
i'd suggest to also add an h2 on how to get the current trace id

amd then have second h2 that explains how to set it with the caverat of this detaching the span from the current spancontext